cmp/tab/TabModel.ts

```diff
@@ -146,16 +164,16 @@ export class TabModel extends HoistModel {
     }
 
     get renderMode(): RenderMode {
-        return this._renderMode ?? this.containerModel.renderMode;
+        return this._renderMode ?? this.containerModel?.renderMode;
```

Comment:
Curious how `containerModel` could be null now. Timing / lifecycle issue when these getters are first read?